node/tests/sync_test.go

```diff
@@ -281,3 +282,40 @@ func TestSyncLightWithTrustedPeers(t *testing.T) {
 
 	assert.EqualValues(t, h.Commit.BlockID.Hash, sw.GetCoreBlockHashByHeight(ctx, 50))
 }
+
+/*
+Test-Case: Broadcast Fraud Proof to different nodes
+Pre-Requisites:
+- CoreClient is started by swamp
+- CoreClient has generated 15 blocks
+Steps:
+1. Create a Bridge Node(BN) with broken block an height 20
+2. Start a BN
+3. Create a Full Node(FN) with a connection to BN as a trusted peer
+4. Start a FN
+5. Check FN is not synced to 21
+*/
+func TestFraudProofBroadcasting(t *testing.T) {
```

Comment:
Let's create a separate test file for fraud proofs as this test is not directly testing sync but rather relies on sync as a pre-req

Reply:
moved to `fraud_test.go`